app/models/park.rb

```diff
@@ -0,0 +1,8 @@
+class Park < ApplicationRecord
+  validates :name, presence: true
```

Comment:
You need to create a unit test for this model. Consider using `shoulda-matchers` to make things a little easier